collections/_services/certbot.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: service
+type: service
+
+title: Certbot
+version: ''
+order: 6
+description: >-
+   Use Certbot to automatically update TLS Certificates on OpenShift Routes.
+descriptionLinks:
+  - link:
+      url: https://github.com/BCDevOps/certbot
+      text: Github
+  - link:
+      url: https://github.com/BCDevOps/certbot/blob/master/README.md
+      text: Review Certbot documentation
+contentStyle: unstructured
+onboard:
+  - 'Self Hosted Service'
+  - 'DockerHub'
+onboardDescription: We provide a <em>Docker Image</em> as well as documented <em>open-source code</em> for hosting your own service.
```

Comment:
instead of '...for hosting your own service' maybe something like:
Deploy on OpenShift using the docker image or create your own image from the source code in GitHub